packages/elements/src/components/app/app.component.ts

```diff
@@ -177,9 +191,43 @@ export class MutationTestReportAppComponent extends LitElement {
   public static styles = [globals, unsafeCSS(theme), tailwind];
 
   public readonly subscriptions: Subscription[] = [];
+
   public connectedCallback() {
     super.connectedCallback();
     this.subscriptions.push(locationChange$.subscribe((path) => (this.path = path)));
+    this.initializeSSE();
+  }
+
+  private source: EventSource | undefined;
+
+  private initializeSSE() {
```

Comment:
```suggestion
  private initializeSse() {
```

I think the casing should be `Sse`. Like `Xml`, `Html`.